src/main/java/decodes/db/Database.java

```diff
@@ -139,8 +139,11 @@ public Database(int type, String location)
 	/**
 	 * Retrieves the 'current' database.
 	 * The 'current' database is set by a previous call to setDb().
+	 * @returns a global instance of the database
+	 * @deprecated since 7.0.9 existing code can use this, but anything totally new that requires the decodes database
+	 * *MUST* take the Database in its constructor. If upgrading/updating things at least consider trying to move
+	 * to constructor injections; even if it means just moving this call up a level.
 	 */
```

Comment:
consider adding @deprecated annotation.

Reply:
Thanks, it showed cross in my IDE so I probably just thought it was there.